autoload/neomake/makers/ft/rust.vim

```diff
@@ -21,6 +21,33 @@ function! neomake#makers#ft#rust#rustc() abort
         \ }
 endfunction
 
+function! neomake#makers#ft#rust#cargotest() abort
+    let maker = {
+        \ 'exe': 'cargo',
+        \ 'args': ['test', '--quiet'],
+        \ 'auto_enabled': 1,
+        \ 'cwd': '%:p:h',
```

Comment:
It this necessary really?
If so, why?
(it gets overwritten below for when a cargo.toml is found anyway - so this is just a bit confusing at least - it should be set in one place)

Reply:
You're right, not necessary. I think it's an artifact for when I tried to test just one file (Neomakemake vs NeomakeProject). But I figured out later that cargo test with a file will do just that.